config/config.go

```diff
@@ -270,3 +273,19 @@ func buildBaseHttpUrl(tlsEnabled bool, hostname string, port int) string {
 
 	return fmt.Sprintf("%s://%s:%d", protocol, hostname, port)
 }
+
+func ConvertConfigtoInternal(config map[string][]string) map[string]map[string]bool {
```

Comment:
I would make this method private to the module.  With go lang, "private" functions begin with a lower case letter.